src/sparseml/exporters/transforms/flatten_qparams.py

```diff
@@ -48,11 +52,12 @@ def transform(self, model: ModelProto) -> ModelProto:
         for init in model.graph.initializer:
             if init.name not in inits_to_flatten:
                 continue
+            _LOGGER.debug(f"Flattening initializer {init.name}")
```

Comment:
note `%` style formatting preferred for debug logs as string formatting is only evaluated on debug call time

Reply:
Are you sure about this? `"%d" % 2` returns a string object, not a special formatting object which makes me think it doesn't actually do lazy evaluation?

Reply:
so you'd do `.debug("foo %s", my_string)`

honestly overhead here is minimal since we're running through once, the main thing that is delayed I believe is calling `str` on the injected variable(s). we take more care with this in say the deepsparse server where we could have lots of concurrent calls running in prod (not debug mode)

Reply:
Ohhhhh didn't realize you could pass format args into debug, nice